CONTRIBUTING.md

```diff
@@ -2,36 +2,43 @@
 
 ### Ways to Contribute
 
-1. Write documentation
-2. Find bugs and create issues
-3. Help code and fix issues
-4. Propose design suggestions and improvements
+1. [Write documentation](#write-documentation)
+2. [Find bugs and create issues](#find-bugs-and-create-issues)
+3. [Help code and fix issues](#help-code-and-fix-issues)
+    1. [Localhost Setup](#localhost-setup)
+    2. [Helphub Theme](#helphub-theme)
+4. [Propose design suggestions and improvements](#propose-design-suggestions-and-improvements)
 
 ----------
 
 ### Write documentation
 There are always teams that need help with writing documentation. The Docs Team can help connect you to the people who need the help. Join our discussions in the [#docs](https://make.wordpress.org/docs/tag/docs/) slack channel on **Thursdays from 17:00 - 18:00 UTC** and offer your help.
 
 ### Find bugs and create issues
-Take a look at the HelpHub site located at [wp-helphub.com](https://wp-helphub.com/). If you see any bugs or issues, please create an issue on our Github repo here: https://github.com/Kenshino/HelpHub/issues. 
+Take a look at the HelpHub site located at [wp-helphub.com](https://wp-helphub.com/). If you see any bugs or issues, please create an issue on our Github repo here: https://github.com/Kenshino/HelpHub/issues.
 
 ### Help code and fix issues
 
+#### Localhost Setup
```

Comment:
_localhost_ has nothing to do with a local install, it's a specific TCP/IP address name, let's use `Local Install` instead, i.e. `#### Local Install`